lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -208,6 +208,7 @@ data ApiErrorCode
     | CannotCoverFee
     | NetworkUnreachable
     | CreatedInvalidTransaction
+    | CannotValidateSelection
```

Comment:
Hmmm. The formulation is a bit misleading here. It's not that we can't validate the selection, it's really that the selection is not valid.  Also, this error code seems redundant with `CreatedInvalidTransaction` (cf other review comment about transaction layer consistency)

Reply:
Agree, `CreatedInvalidTransaction` should be reused here